### PR TITLE
[WIP] Convert CVSSv2 into CVSSv3

### PR DESCRIPTION
**Pull request by ziadhany**
issue : #1187

### CODE REVIEW

**Comment by ziadhany** [comment]
according to my assumption, I tried to find the best way of doing this. 
but I am not sure about how accurate the result 

Have a look at the result : [output.txt](https://github.com/nexB/vulnerablecode/files/11599822/output.txt)

File: vulnerabilities/severity_systems.py [hunk 1]
```diff
@@ -161,3 +161,121 @@ def compute(self, scoring_elements: str) -> str:
         APACHE_TOMCAT,
     )
 }
+
+
+def convert_cvssv2_vector_to_cvssv3_vector(cvssv2_vector):
```
Comment (pombredanne) [inline vulnerabilities/severity_systems.py:166]: Can you extract this function in a seperate file?

Comment (ziadhany) [reply vulnerabilities/severity_systems.py]: Done

**Comment by ziadhany** [comment]
Test result :

Screenshot from 2023-07-25 16-11-46

**Comment by pombredanne** [comment]
@ziadhany re: https://github.com/nexB/vulnerablecode/pull/1204#issuecomment-1649844996
Does this mean that mosts v2 to v3 tests are failing ... compared to the NVD?

**Comment by TG1999** [comment]
@ziadhany @pombredanne what shall be done for this PR ?

**Pull request closed without merging**